packages/plugin-assets-retry/src/index.ts

```diff
@@ -39,6 +40,11 @@ export const pluginAssetsRetry = (
           HtmlPlugin,
         },
       ]);
+
+      const isRspack = api.context.bundlerType === 'rspack';
+      chain
```

Comment:
Maybe one plugin, but AsyncRetryPlugin is aligned with https://github.com/mattlewis92/webpack-retry-chunk-load-plugin, If it is to be exported separately in the future, the current structure is better.